File: CSharp/1794.cs
using System;
using System.Linq;

class Program
{
    static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        int[] lava = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int[] seca = Console.ReadLine().Split().Select(int.Parse).ToArray();

        if(n>= lava[0] && n<=lava[1] && n>=seca[0] && n<=seca[1])
        {
            Console.WriteLine("possivel");
        }
        else{Console.WriteLine("impossivel");}

    }
}

File: Python/1794.py
n = int(input())
lava = list(map(int, input().split()))
seca = list(map(int, input().split()))


if lava[0] <= n <= lava[1] and seca[0] <= n <= seca[1]:
    print("possivel")
else:
    print("impossivel")
